README.md

```diff
@@ -16,6 +16,12 @@ pip install mozanalysis
 
 ## Testing locally
 
+### with Tox
+
+Install tox into your global Python environment and run `tox`.
```

Comment:
Might be friendly to point out that while they're hacking, people can run a single file of tests by something like:
```
tox -e py37 tests/stats/test_bootstrap.py
```

Reply:
Added a couple examples.